backend/app/models/batch_job.py

```
@@ -0,0 +1,122 @@
+    organization_id: int = Field(foreign_key="organization.id")
+    project_id: int = Field(foreign_key="project.id")
+
+    # Timestamps
+    inserted_at: datetime = Field(default_factory=datetime.utcnow)
+    updated_at: datetime = Field(default_factory=datetime.utcnow)
+
+    # Relationships
+    organization: Optional["Organization"] = Relationship(  # noqa: F821
+        back_populates="batch_jobs"
+    )
+    project: Optional["Project"] = Relationship(
```

Ruff (0.14.2)

Undefined name `Project`

(F821)